### DISCUSSION TITLE
[@types/memory-cache] del has wrong return type

### DISCUSSION THREAD

**Discussion by sosoxuc** [DefinitelyTyped/DefinitelyTyped#65619] [Issues with a @types package]
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/memory-cache/index.d.ts#L12

[@types/memory-cache] `del` has wrong return type

Should not `del` has boolean return type?

[a comment by an automated account is not shown]

**Comment by thieman** [comment]
I agree that `del` should return a boolean, see the implementation for it here: https://github.com/ptarjan/node-cache/blob/master/index.js#L47-L65

Converted this discussion to an issue, #65622.

**Reply by thieman** [reply]
Fixed by https://github.com/DefinitelyTyped/DefinitelyTyped/pull/65623.
